modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordApi.kt

```diff
@@ -270,8 +270,10 @@ class MirrordApi(private val service: MirrordProjectService) {
 
 /**
  * A mirrord CLI invocation.
+ *
+ * @param args: An extra list of arguments (used by `verify-config`).
  */
-private abstract class MirrordCliTask<T>(private val cli: String, private val command: String, private val args: String?) {
+private abstract class MirrordCliTask<T>(private val cli: String, private val command: String, private val args: Map<String, String>?) {
     var target: String? = null
```

Comment:
okay i think wouldnt it be better to just pass a String? creating a map is space extensive and probably un needed in this case?

Reply:
Changed to a list, when I tried just passing a string in the implementation method it did not work quite as expected?

Reply:
does it work for you locally?